tasks.js

```diff
@@ -0,0 +1,79 @@
+class newTask {
```

Comment:
OOP gibt es zwar in JavaScript, wird aber von den meisten Entwicklern als unnötige "Boilerplate" angesehen. Was du hier erreichen willst, ist Typsicherheit, da kommen wir mit TypeScript hin. 😉